cloud-minecraft/cloud-minecraft-extras/src/main/java/cloud/commandframework/minecraft/extras/MinecraftHelp.java

```diff
@@ -439,9 +447,21 @@ private void printVerboseHelpTopic(
                 .append(text(": ", this.colors.primary))
                 .append(this.highlight(text("/" + command, this.colors.highlight)))
         );
-        final Component topicDescription = helpTopic.getDescription().isEmpty()
-                ? this.messageProvider.provide(sender, MESSAGE_NO_DESCRIPTION)
-                : this.descriptionDecorator.apply(helpTopic.getDescription());
+        /* Topics will use the long description if available, but fall back to the short description. */
+        final Component richDescription =
+                helpTopic.getCommand().getCommandMeta().get(MinecraftExtrasMetaKeys.LONG_DESCRIPTION)
+                        .orElse(helpTopic.getCommand().getCommandMeta().get(MinecraftExtrasMetaKeys.DESCRIPTION)
+                                .orElse(Component.empty()));
+
+        final Component topicDescription;
+        if (richDescription != Component.empty()) {
```

Comment:
Is there any benefit to using `Component.empty()` instead of just `null` here?

Reply:
uh not really, it's just habit to use the empty component rather than null since that's what we do in a lot of places in Adventure.